packages/utils/tags/index.js

```diff
@@ -0,0 +1,2 @@
+module.exports.html = new Set(require('html-tags'));
+module.exports.svg = new Set(require('svg-tag-names'));
```

Comment:
I thing it might be useful to keep the Set of all tags by default, like

```js
const html = new Set(require('html-tags'))
const svg = new Set(require('svg-tag-names'))

module.exports = new Set([...html, ...svg])
module.exports.html = html
module.exports.svg = svg
```

because in most of the cases I think it might be easier to use just `tags.has`, when it does not matter what kind of tags it is

and have you checked if there is no usage of `tags` in the code? otherwise the current changes might break something

Reply:
fixed